docs/src/formulation-details.md

```diff
@@ -1,46 +1,64 @@
-# Three-phase formulation details
+# Formulation details
 
+```@meta
+CurrentModule = ThreePhasePowerModels
+```
 
-## `StandardACPForm`
-Real-valued formulation from:
-- Formulation without shunts: Mahdad, B., Bouktir, T., & Srairi, K. (2006). A three-phase power flow modelization: a tool for optimal location and control of FACTS devices in unbalanced power systems. In IEEE Industrial Electronics IECON (pp. 2238–2243).
+## `ACPPowerModel`
+Real-valued formulation from (extended with shunts):
+```
+@INPROCEEDINGS{4153495,
+author={B. Mahdad and T. Bouktir and K. Srairi},
```

Comment:
@ccoffrin, do you have a better / older reference for this? Preferably one with the shunts already included?

Furthermore, we can't really use a call to 
```
@docs
ACPPowerModel
```
here, as this would collect the reference for the balanced formulation.